manifests/client.pp

```diff
@@ -347,6 +374,13 @@
     ],
   }
 
+  file { "${etc_directory}/openvpn/${server}/download-configs/${name}.tar.gz":
+    ensure  => present,
+    replace => 'no',
+    content => "not yet created\n",
```

Comment:
why is there a content attribute for a .tar.gz file?

Reply:
It works also without. I wanted to have the possibility to know if something was wrong with content of file.

content removed.